web/config/env.js

```diff
@@ -2,6 +2,7 @@
 const fs = require('fs');
 const path = require('path');
 const paths = require('./paths');
+const { name, version } = require('../../package.json')
```

Comment:
Si tu veux te passer du commentaire : 
```suggestion
const { name as envName, version as appVersion } = require('../../package.json')
```

Reply:
Si on laisse ainsi, c'est générique avec webapp v1 et pro

Reply:
ok pour moi